tests/unit/test_normalize_icon_type.py

```diff
@@ -0,0 +1,65 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2005-2022, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License (version 2
+# or later) with exception for distributing the bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#
+# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
+#-----------------------------------------------------------------------------
+
+import os
+import sys
+
+import pytest
+
+
+def test_normalize_icon_type(monkeypatch):
+    from PyInstaller.building.icon import normalize_icon_type
+
+    # Currently assumes PIL/Pillow is installed on the testing machine
```

Comment:
```suggestion
    pytest.importorskip("PIL")
```
Not any more!

Reply:
I put it down a bit so it could still fail earlier on the tests that don't need PIL.